visivo/parsers/evaluator.py

```diff
@@ -0,0 +1,143 @@
+from typing import Any
+import ast
+import operator as op
+import os
+
+SUPPORTED_NUMPY_FUNCTIONS = ["sum", "all", "mean"]
```

Comment:
@tgsoverly why not allow all numpy functions? Are you thinking that would be dangerous?

Reply:
I want to make sure they work, and have a test for them.  Easy to add, we can add a bunch of them.